pages/createOrder.js

```diff
@@ -230,6 +230,10 @@ const collectiveFields = `
     id
     name
     settings
+    connectedAccounts {
+      id
+      service
+    }
```

Comment:
It's a good way to solve it for now, in the future a `supportsPaypal` field could be nice. I've updated https://github.com/opencollective/opencollective/issues/2807 which reflects similar needs to eventually cover that in the future.